packages/front-end/components/Features/FeatureModal/index.tsx

```diff
@@ -212,6 +229,19 @@ export default function FeatureModal({
     >
       <FeatureKeyField keyField={form.register("id")} />
 
+      {!featureToDuplicate && (
+        <div className="form-group">
+          <label>Project</label>
+          <SelectField
+            value={form.watch("project")}
+            onChange={(p) => form.setValue("project", p)}
+            name="project"
+            initialOption={allowAllProjects ? "All Projects" : undefined}
+            options={availableProjects}
+          />
+        </div>
+      )}
```

Comment:
A couple comments:
1. We should only show this if there are available projects to choose from.  Also, I think it's best to only show if "All Projects" is selected in the left nav.  If you have Project A selected in the left nav, it seems weird to be able to create features in Project B
2. It's fine to show this when duplicating features (as long as the other conditions above are met)
3. The SelectField component has support for rendering labels and form-group divs already
4. The `initialOption` text should always just be "None"

```suggestion
      {!project && availableProjects.length > 0 && (
        <SelectField
          label="Project"
          value={form.watch("project")}
          onChange={(p) => form.setValue("project", p)}
          initialOption="None"
          options={availableProjects}
        />
      )}
```